vicodyn/src/vicodyn/proxy.cpp

```diff
@@ -448,6 +518,7 @@ proxy_t::proxy_t(context_t& context, asio::io_service& loop, peers_t& peers, con
     peers(peers),
     app_name(name.substr(sizeof("virtual::") - 1)),
     balancer(make_balancer(args, extra)),
+    max_buffer_size(args.as_object().at("max-buffer-size-kb", 10240U).as_uint()),
```

Comment:
Let's stick to use `_` instead of `-` in config parameters' names. As I found `_` is more widely used.

Reply:
Ok